komodo/show_version.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+import argparse
+import configparser
+import os
```

Comment:
I tend to prefer not to import individual functions, as it makes it more readable when that function is in use further down in the code (then I don't have to look at the top to see where that function comes from). But that might come at a speed penalty?

Reply:
That is indeed more readable to have the full path. 
I agree that the downside can be speed. Do you think this would make any difference in terms of security? 

There is a way to import one individual function and keep readability (from python documentation):
"""
Users of the package can import individual modules from the package, for example:

import sound.effects.echo
This loads the submodule sound.effects.echo. It must be referenced with its full name.

sound.effects.echo.echofilter(input, output, delay=0.7, atten=4)
"""

How does it look for you?

Reply:
I'm never sure which way to go with this, but in general I think I prefer more namespace. I don't think it's a speed thing: the entire module is evaluated either way.

It's true you can import submodules with that other pattern, so `import os.path` would be okay, but there's no other way to import `argparse.ArgumentParser` for example. I'm not using any submodules so I'm not sure there's much to do here.